packages/compose/package.json

```diff
@@ -38,7 +38,7 @@
 		"@wordpress/keycodes": "file:../keycodes",
 		"@wordpress/priority-queue": "file:../priority-queue",
 		"change-case": "^4.1.2",
-		"clipboard": "^2.0.8",
+		"clipboard": "^2.0.11",
```

Comment:
We need to update the version of clipboard we are using so that the `.copy` function is available